Flutter/treemap/legend.md

```diff
@@ -204,65 +274,120 @@ class SocialMediaUsers {
 {% endhighlight %}
 {% endtabs %}
 
-![legend icon color](images/legend/icon-color.png)
+![First label customization](images/legend/first-label-customization.png)
 
-## First segment label customization
+## Should always show scrollbar
 
-You can customize the first segment label of the legend using the [`TreemapColorMapper.name`](https://pub.dev/documentation/syncfusion_flutter_treemap/latest/treemap/TreemapColorMapper/name.html) property with curly braces. The first curly brace value will be applied as segment start label and the next curly brace value will be applied as segment end label. By default, the [`TreemapColorMapper.from`](https://pub.dev/documentation/syncfusion_flutter_treemap/latest/treemap/TreemapColorMapper/from.html) value is placed at the starting position of first segment and the [`TreemapColorMapper.to`](https://pub.dev/documentation/syncfusion_flutter_treemap/latest/treemap/TreemapColorMapper/to.html) value is placed at the ending position of the first segment.
+You can control the visibility of the scrollbar in the treemap using the [`shouldAlwaysShowScrollbar`](https://pub.dev/documentation/syncfusion_flutter_treemap/latest/treemap/TreemapLegend/shouldAlwaysShowScrollbar.html) property. When set to true, the scrollbar will always be visible, allowing easy navigation through the treemap content even when content is fully within view. The default value is `false`, meaning the scrollbar will only appear when the content is scrollable.

```

Comment:
When set to true -> 'true' not highlighted

Reply:
allowing easy navigation through the treemap content even when content is fully within view - Conveys wrong meaning. It seems like treemap tiles scrolling and not like legend item scrolling